e2e/node/src/node.test.ts

```diff
@@ -185,6 +185,9 @@ describe('Build Node apps', () => {
         dependencies: {
           '@nestjs/common': '^7.0.0',
           '@nestjs/core': '^7.0.0',
+          '@nestjs/platform-express': '^7.0.0',
+          'reflect-metadata': '^0.1.13',
+          rxjs: '~6.6.3',
```

Comment:
these dependencies are now detected, as they're peer dependencies of dependencies that are used by the application code